website/assets/styles/pages/capex-savings.less

```
@@ -0,0 +1,514 @@
+#capex-savings {
+  @heading-lineheight: 120%;
+  @text-lineheight: 150%;
+  background: linear-gradient(180deg, #E8F1F6 0%, #FFF 2.51%);
+  //…
+  h1 {
+    .page-heading();
```

Stylelint (17.14.1)

[error] Unknown word .page-heading (CssSyntaxError)

(CssSyntaxError)